src/main/features/claimant-response/routes/part-payment-received.ts

```diff
@@ -0,0 +1,64 @@
+import * as express from 'express'
+import { Paths } from 'claimant-response/paths'
+import { ErrorHandling } from 'main/common/errorHandling'
+import { Claim } from 'main/app/claims/models/claim'
+import { Form } from 'main/app/forms/form'
+import { FormValidator } from 'main/app/forms/validation/formValidator'
+import { Draft } from '@hmcts/draft-store-client'
+import { DraftService } from 'services/draftService'
+import { YesNoOption } from 'main/app/models/yesNoOption'
+import { PartPaymentReceived } from 'claimant-response/form/models/states-paid/partPaymentReceived'
+import { DraftClaimantResponse } from 'claimant-response/draft/draftClaimantResponse'
+import { StatesPaidHelper } from 'claimant-response/helpers/statesPaidHelper'
+import { GuardFactory } from 'response/guards/guardFactory'
+import { NotFoundError } from 'main/errors'
+import { Logger } from '@hmcts/nodejs-logging'
+
+const stateGuardRequestHandler: express.RequestHandler = GuardFactory.create((res: express.Response): boolean => {
```

Comment:
Please refactor out under `src/main/features/claimant-response/guards`, we can add unit tests as well

Reply:
This is still pending.